README.md

```diff
@@ -1,69 +1,28 @@
-
-Q: Can I use transpilers?  
-A: Yes, but you [may not need to](http://kangax.github.io/compat-table/es6/).
-
-Q: Can I use libraries, frameworks, etc?  
-A: Yes, but please consider the 'What we are looking for?' guidelines.
-
-Q: Should my solution be client-side only/server-side only/client-server?  
-A: That's up to you.  We are interested in how you solve the problem, so we don't want to force you down any particular route.
-
-## Option 2
-
-If you have some personal code that you would be prepared to share with us, we can assess that instead.  The code should meet the following criteria:
-
-- It should be at least 1-2 hours of your own work
-- Ideally, it should involve an element of web development
-- It should demonstrate how you approach a problem
-- It should be something that you are able to discuss with us
-
-### Submission
-
-You can submit your code however you want - send the details to your point of contact.  Please include a README containing any setup instructions and keep the setup steps simple.
-
-## Option 3
-
-Create a node.js 'Hello World' web application and deploy it to a free AWS account using appropriate tools and best practices.  When deployed, the root url should display 'Hello World'. Do not send us your AWS credentials - we will take your solution and deploy it to our own account.
-
-### What we are looking for
-
-- We anticipate that the app may have future infrastructure requirements, so you should aim for maintainability with your solution.
-- Those naughty [Chaos Monkeys](https://github.com/netflix/chaosmonkey) have been known to randomly terminate AWS instances.  Your solution should be resilient to this.
-- Members of the team use various OSes and distributions.  It would be great if they could develop the application locally in a consistent environment.
-- We want to discover how you handle infrastructure day-to-day.
-- Monitoring is great but we don't want you to worry about it for now.
-- The same goes for CI servers.
-
-## Submission
-
-You can submit your code however you want - send the details to your point of contact.  Please include a README containing any setup instructions and keep the setup steps simple.
+# Overview
+
+This is a coding exercise to help us assess candidates looking to join the team at Condé Nast International.  The test is intended to be completed at home, and we expect you to spend no more than half a day working it out. We appreciate the time you spent on it and will having it examined by one of our engineer, we will go through your submitted work together when you're invited to meet us face-to-face.        
+
+## Which test shall I pick?
+ - [Software Engineer](se-test)
+ - [Infrastructure Engineer](infra-test)   
+
+## How to submit code
+We prefer to see your work as a GIT repo with commits from which the app is built up.
+You can submit by sending up the link to a public git repository on Github, or if for some reason you'd 
```

Comment:
```suggestion
You can submit by sending up the link to a public git repository on Github, or if you'd 
```
The "if for some reason" could come across passive-aggressive, or at least implies we would think negatively of you if you provided us with this.

I know this was not your intent, though! Thoughts?